src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  width: calc($cell-size * 7 + $gap * 6);
+  padding: 10px;
+
+  &__day {
+    box-sizing: border-box;
+
+    width: $cell-size;
```

Comment:
The variable '$cell-size' is used here, but it is not defined within this file. Ensure that '$cell-size' is defined in this file or imported from another SCSS file.